src/dev/beta.ts

```diff
@@ -0,0 +1,169 @@
+/*!
```

Comment:
Let's mention this in CONTRIBUTING.md : https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#tooling